Ginger/Ginger/BusinessFlowPages_New/VariabelsListViewPage.xaml.cs

```diff
@@ -223,7 +223,18 @@ private void ListVars_PreviewDragItem(object sender, EventArgs e)
             if (DragDrop2.DragInfo.DataIsAssignableToType(typeof(VariableBase)))
             {
                 // OK to drop                         
-                DragDrop2.DragInfo.DragIcon = GingerWPF.DragDropLib.DragInfo.eDragIcon.Copy;
+                if ((sender as DragInfo).DragSource == (sender as DragInfo).DragTarget)
```

Comment:
Can't we do the move scanrio handling automatic?
meaning:
if we identify that the drag is on same list so automatically we will set the icon to "Move" and allow it
if we identify that drag is on other list so we will invok the approval of the specific page to update if can drop on it or not and based on that we will set the icon of "Copy" (which needs to be modified to "Add") or "DoNotDrop"

Reply:
Yes. That's the far better option. I'll update & push the same.